addons/account/models/account_report.py

```diff
@@ -712,7 +712,7 @@ def write(self, vals):
 
     def copy_data(self, default=None):
         vals_list = super().copy_data(default=default)
-        return [dict(vals, label=self.env._("%s (copy)", expression.label)) for expression, vals in zip(self, vals_list)]
+        return [dict(vals, label=expression.label) for expression, vals in zip(self, vals_list)]
```

Comment:
Is it not equivalent to simply remove this function override and add "copy=True" to the definition of the "label" field ?